doc/installation.rst

```diff
@@ -43,14 +43,15 @@ twice to get the very latest) to ensure you have the latest version::
    > conda update -n base conda
    > conda update -n base conda
 
-You should then make sure you have `anaconda-project` installed::
+We recommend installing `anaconda-project` is a new environment, to keep
+things separate and clean. We'll create an environment called `project`, 
+please select another name if you already have an environment named this way::
 
-   > conda install anaconda-project
+   > conda create -n project "anaconda-project>=0.11"
```

Comment:
I'm not sure I agree with this step. I agree that people shouldn't normally install into their base environment, but we've never told anyone to do that. If they are defaulting to installing into base, I don't think it's up to us to worry about that; they'll have their own problems with that, and it's unlikely that anaconda-project would be the culprit. In fact, if there is anything that _should_ be in a base environment, it's meta-utilities like anaconda-project, which enable working with other environments. Rather than get into any of that, I'd rather keep these instructions simple and have people just install anaconda-project wherever it's convenient for them.

Reply:
I remember suggesting that change in a discussion and having no approval/no objection, and I'm not the one running the tutorial, so I'm fine reverting!

For reference, this change was motivated by someone I helped during the tutorial. That person had a broken *base* environment and basically the only solution was to create a new environment to install anaconda-project. It happens that it's also my current setup.

> it's meta-utilities like anaconda-project

As far as I know there's no concept of meta utilities in conda. I'd love to have something like `pipx` - that is now pretty popular in the pip world - to easily install utilities in their own environment and have them added to PATH.

> it's unlikely that anaconda-project would be the culprit

True. Yet it does come with a number of dependencies.

> I agree that people shouldn't normally install into their base environment, but we've never told anyone to do that

Well we don't tell them to activate any other environment, so they're very likely to install `anaconda-project` in their base environment. After all the tutorial is given by Anaconda employees, they have no reason not to trust us on our install procedures and decide to add an extra step. Probably some people in the room have heard that it's not good practice to install packages in their base environment, yet I'd bet they will still do it. Not providing more details on where and how to install anaconda-project assumes that people know what they're doing when they run `conda install`, that's a big assumption :)

I think an interesting effect of this change is that it leaves the system of those who follow the tutorial in a state that easily allows them to get back to their initial state. They can just decide to remove the environments that have been created; how to clean up what has been installed/downloaded  should actually be documented somewhere in the tutorial 🙃 Currently they'd have to remove anaconda-project from *base*, in all likelihood. Attempting to remove anything from *base* gives me goosebumps!